backend/app.js

```diff
@@ -40,7 +40,7 @@ app.get("/api/v2/lumens/total-supply", lumensV2.totalSupplyHandler);
 app.get("/api/v2/lumens/circulating-supply", lumensV2.circulatingSupplyHandler);
 
 app.get("/api/v3/lumens", lumensV3.handler);
-app.get("/api/v3/lumens/total-supply-check", lumensV3.totalSupplySumHandler);
+app.get("/api/v3/lumens/total-supply-check", lumensV3.totalSupplyCheckHandler);
 /* For CoinMarketCap */
 app.get("/api/v3/lumens/total-supply", lumensV3.totalSupplyHandler);
 app.get("/api/v3/lumens/circulating-supply", lumensV3.circulatingSupplyHandler);
```

Comment:
💡 I think we can remove these two endpoints and as part of `v3` deprecate the old v2 endpoints and tell people to use the new endpoint. The new endpoint above contains all the same data and if we're introducing a new version of the API we can deprecate duplicate data points. Thoughts @acharb @accordeiro?

Reply:
makes sense to me. I'm not sure why we had the 2 single endpoints for CoinMarketCap in v2 though, since the `api/v2/lumens` also had that data too. I'll create a ticket to track that. In the mean time for this PR can we keep to debug the runscope error?